staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -329,6 +330,19 @@ func (c *Cacher) RequestWatchProgress(ctx context.Context) error {
 	return c.storage.RequestWatchProgress(ctx)
 }
 
+func (c *Cacher) Supports(ctx context.Context, features ...featuregate.Feature) (bool, error) {
+	for _, feature := range features {
+		supported, err := c.storage.Supports(ctx, feature)
```

Comment:
It depends on the features, but I can definitely imagine features that can actually be implemented in watchcache layer).
For now this is fine, but some TODO to extend this implementation to support cacher feature-checker would probably be useful.